staging/src/k8s.io/cri-api/pkg/apis/runtime/v1/api.proto

```diff
@@ -445,6 +445,7 @@ message PodIP{
     string ip = 1;
 }
 // PodSandboxNetworkStatus is the status of the network for a PodSandbox.
+// Ignored for pods sharing the host networking namespace.
```

Comment:
@mrunalp you can find the context in this thread https://github.com/kubernetes/kubernetes/pull/106715/files#r758454031